content/en-US/api/textWordSpacing.md

```diff
@@ -0,0 +1,51 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.textWordSpacing'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/textWordSpacing
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.textWordSpacing
+---
+
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/textWordSpacing](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/textWordSpacing).**
+
+## Description
+
+The `textWordSpacing` property of the `CanvasRenderingContext2D` interface
+returns a double which represents horizontal spacing behavior between text
+words. Setting `textWordSpacing` to postive values causes words to
+spread farther apart, while negative value brings them closer together.
```

Comment:
value -> values

Reply:
make sense, positive values and negative value don't match.